### PR TITLE
Update storage format docs

### PR DESCRIPTION
**Pull request by kolesnikovae**
A tiny update of the block format description. More detailed documentation will come later.

Depends on #2305 (I'll rebase after it is merged)

### CODE REVIEW

File: docs/sources/reference-pyroscope-architecture/block-format/_index.md [hunk 1]
```diff
@@ -19,25 +19,24 @@ the block there are multiple files:
...
-  Prometeus'/Loki's label and typical labels are `namespace` and `pod` to
-  describe which workload the profiles are coming form.
+definition][pprof] for the pprof wire format.
 
-* Pprof Labels: Part of the pprof payload and used by the agent or the software
-  to transport additional information.
+Profile labels contain additional information gathered at ingestion time and can
+be used to select certain profiles. They are comparable to Prometeus'/Loki's label
+and typical labels are `namespace` and `pod` to describe which workload the profiles
+are coming form.
```
Comment (kolesnikovae) [inline docs/sources/reference-pyroscope-architecture/block-format/_index.md:39]: We no longer store pprof labels

File: docs/sources/reference-pyroscope-architecture/block-format/_index.md [hunk 2]
```diff
@@ -19,25 +19,24 @@ the block there are multiple files:
...
-
-* External Labels: Contain additional information gathered at ingestion time
-  and can be used to select certain profiles. They are comparable to
-  Prometeus'/Loki's label and typical labels are `namespace` and `pod` to
-  describe which workload the profiles are coming form.
+definition][pprof] for the pprof wire format.
 
-* Pprof Labels: Part of the pprof payload and used by the agent or the software
-  to transport additional information.
+Profile labels contain additional information gathered at ingestion time and can
```
Comment (simonswine) [inline docs/sources/reference-pyroscope-architecture/block-format/_index.md:36]: Maybe that's worth it?
```suggestion
Profile series labels contain additional information gathered at ingestion time and can
```

File: docs/sources/reference-pyroscope-architecture/block-format/_index.md [hunk 3]
```diff
@@ -19,25 +19,24 @@ the block there are multiple files:
...
+be used to select certain profiles. They are comparable to Prometeus'/Loki's label
```
Comment (simonswine) [inline docs/sources/reference-pyroscope-architecture/block-format/_index.md:37]:
```suggestion
be used to select certain profiles. They are comparable to Prometeus'/Loki's labels
```

File: docs/sources/reference-pyroscope-architecture/block-format/_index.md [hunk 4]
```diff
@@ -19,25 +19,24 @@ the block there are multiple files:
...
+and typical labels are `namespace` and `pod` to describe which workload the profiles
```
Comment (simonswine) [inline docs/sources/reference-pyroscope-architecture/block-format/_index.md:38]:
```suggestion
and typical label names are `namespace` and `pod` to describe which workload the profiles
```

File: docs/sources/reference-pyroscope-architecture/block-format/_index.md (hunk 1 above)
Comment (simonswine) [inline docs/sources/reference-pyroscope-architecture/block-format/_index.md:39]:
```suggestion
are coming from.
```

**Review by simonswine — APPROVED** [review]
Commented on a few things, but in general LGTM

**Review by bryanhuhta — APPROVED** [review]

**Pull request merged**